app/translations/ga/LC_MESSAGES/messages.po

```diff
@@ -37,157 +38,164 @@ msgstr[2] "%(num)s mí"
 msgstr[3] "%(num)s míonna"
 msgstr[4] "%(num)s míonna"
 
-#: app/jinja_filters.py:130
+#: app/jinja_filters.py:137
 #, python-format
 msgid "%(date)s at %(time)s"
 msgstr "%(date)s ag %(time)s"
 
-#: app/jinja_filters.py:141
+#: app/jinja_filters.py:148
 #, python-format
 msgid "%(from_date)s to %(to_date)s"
 msgstr "%(from_date)s go %(to_date)s"
 
-#: app/forms/custom_fields.py:130
-msgid "Not a valid choice"
-msgstr "Ní rogha bhailí í seo"
+#: app/forms/error_messages.py:11 app/forms/error_messages.py:12
+#: app/forms/error_messages.py:13 app/forms/error_messages.py:14
+msgid "Enter an answer"
+msgstr "Iontráil freagra"
+
+#: app/forms/error_messages.py:15
+#, python-format
+msgid "Select an answer <span class=\"u-vh\">to ‘%(question_title)s’</span>"
+msgstr "Roghnaigh freagra<span class=\"u-vh\">ar ‘%(question_title)s’</span>"
 
-#: app/forms/fields.py:231
+#: app/forms/error_messages.py:18
+#: app/forms/field_handlers/dropdown_handler.py:11
 msgid "Select an answer"
 msgstr "Roghnaigh freagra"
 
-#: app/helpers/template_helper.py:19
-msgid "Office for National Statistics logo"
-msgstr "Lógó Oifig an Staidrimh Náisiúnta"
-
-#: app/helpers/template_helper.py:28 app/helpers/template_helper.py:36
-msgid "Census 2021"
-msgstr "Daonáireamh 2021"
-
-#: app/helpers/template_helper.py:32
-msgid "Northern Ireland Statistics and Research Agency logo"
-msgstr "Lógó na Gníomhaireachta um Staidreamh agus Thaighde Thuaisceart Éireann"
+#: app/forms/error_messages.py:19
+#, python-format
+msgid "Select at least one answer <span class=\"u-vh\">to ‘%(question_title)s’</span>"
+msgstr "Roghnaigh freagra amháin ar a laghad<span class=\"u-vh\">ar ‘%(question_title)s’</span>"
```

Comment:
Is there supposed to be a space before <span?

Reply:
It's fixed in crowdin now and added to this PR, thanks!